rites/roman1969/src/calendars/countries/hong-kong/index.ts

```diff
@@ -0,0 +1,81 @@
+import { PatronTitle } from '../../../constants/martyrology-metadata';
+import { Precedences } from '../../../constants/precedences';
+import { CalendarDef } from '../../../models/calendar-def';
+import { Inputs } from '../../../types/calendar-def';
+
+export class HongKong extends CalendarDef {
+  particularConfig = {
+    ascensionOnSunday: true,
+    epiphanyOnSunday: true,
+    corpusChristiOnSunday: true,
+  };
+
+  inputs: Inputs = {
+    // src: http://catholic-dlc.org.hk/English_Liturgy.htm
+    mary_mother_of_god: {
+      isHolyDayOfObligation: false,
+    },
+
+    // src: http://catholic-dlc.org.hk/English_Liturgy.htm
+    joseph_freinademetz_priest: {
+      precedence: Precedences.ProperMemorial_11b,
+      dateDef: { month: 1, date: 29 },
+    },
+
+    // src: http://catholic-dlc.org.hk/English_Liturgy.htm
+    gabriele_maria_allegra_priest: {
+      precedence: Precedences.ProperMemorial_11b,
+      dateDef: { month: 1, date: 30 },
+    },
+
+    // src: http://catholic-dlc.org.hk/English_Liturgy.htm
+    aloysius_versiglia_bishop_and_callistus_caravario_priest_martyrs: {
+      precedence: Precedences.OptionalMemorial_12,
+      dateDef: { month: 2, date: 25 },
+      martyrology: ['aloysius_versiglia_bishop', 'callistus_caravario_priest'],
+    },
```

Comment:
i'm not seeing this entry in the source provided

Reply:
@TobiTenno, it is in [here](http://catholic-dlc.org.hk/sup_massindex.htm#:~:text=Versiglia%20%26%20Callistus%20Caravario):

```suggestion
    // src: http://catholic-dlc.org.hk/sup_massindex.htm#:~:text=%E8%81%96%E9%9B%B7%E9%B3%B4%E9%81%93%E4%B8%BB%E6%95%99%E5%8F%8A%E8%81%96%E9%AB%98%E6%83%A0%E9%BB%8E%E5%8F%B8%E9%90%B8%EF%BC%88Ss%2E%20Louis%20Versiglia%20%26%20Callistus%20Caravario%20%EF%BC%89
    aloysius_versiglia_bishop_and_callistus_caravario_priest_martyrs: {
      precedence: Precedences.OptionalMemorial_12,
      dateDef: { month: 2, date: 25 },
      martyrology: ['aloysius_versiglia_bishop', 'callistus_caravario_priest'],
    },
```

Reply:
I cano only find this in Chinese: http://catholic-dlc.org.hk/sup_massindex.htm. will update the source.